### DISCUSSION TITLE
cura  5.X  freeze when I use octoprint plugin

### DISCUSSION THREAD

**Discussion by jiemeb** [fieldOfView/Cura-OctoPrintPlugin#302] [Q&A]
Hi, 
I use Plugin octoprint with CURA 4.X with no problem. But when I try CURA 5.0 or 5.2.1 when i go to survey application or start printing with octoprint ,CURA  freeze.
I don't have problem to install plugin, get api key, connect to my octoprint. 
If I start printing with CURA  4,  I can survey printing with CURA 5 since printing is on. when printing is finish CURA 5 freeze.
does some body have an idea ?

I try plugin 3.7.0 3.72 3.7.3 dev plugin.
my octoprint is 1.8.6 on bananaPI
JM

**Comment by fieldOfView** [comment]
Perhaps the logs will give me a clue. Cura.log can be found via Help -> Show configuration folder.

**Reply by jiemeb** [reply]
[stderr.log](https://github.com/fieldOfView/Cura-OctoPrintPlugin/files/10216163/stderr.log)
[cura.log](https://github.com/fieldOfView/Cura-OctoPrintPlugin/files/10216164/cura.log)
Find here log file and error for CURA 5.1.2.
thank for tour time/
Regards
JM

**Comment by jiemeb** [comment]
[cura.log](https://github.com/fieldOfView/Cura-OctoPrintPlugin/files/10207035/cura.log)
Thanks for your reply
Here log for Cura 5.0
I can't  get log for 5.2 because I am printing .

**Reply by fieldOfView** [reply]
Unfortunately there's nothing indicating where/why (or even whether) the freeze happens in the logs. That means the freeze happens outside my code which makes it hard to track down.

Cura 5 and newer use a newer version of the Qt framework. With new versions of frameworks come new bugs in those frameworks. At this point, I don't have time to investigate. I will try to revisit this in the new year.

**Reply by jiemeb** [reply]
Thank's for your time. 
I got 5.0 working in the past, But after remove ~/.local/share/cura (because 5.2.1 not working ) only 4.x work.
I thinks problem in some data missing or to be to old in configuration.
mery Xmas for you and your family ;-)
JMM

**Reply by jiemeb** [reply]
Now plugin is working. I found the solution. 
remove all config gor CURA on ~/.config/cura
```
rm -rf ~/.config/cura
rm -rf ~/.local/share/cura
```
and after that. install CURA and the plugin.

Thank's for your help.
JM

**Comment by jiemeb** [comment]
octoprint plugin freeze  again.
When in ~/config/cura/5.2/cura.cfg have `window_height = 938` cura freeze when I select monitor.
see attached files 
[cura.zip](https://github.com/fieldOfView/Cura-OctoPrintPlugin/files/10352188/cura.zip)

**Comment by jiemeb** [comment]
When in ~/config/cura/5.2/cura.cfg have window_height = 912 or more cura freeze when I select monitor.
working below 912.

**Reply by fieldOfView** [reply]
That is an "interesting" discovery. What resolution is you display? Does the *width* of the window play any role?

You *could* uncheck the option "restore window position on start" in the General preferences. That way Cura will not read the window_height value from the configuration file.

**Comment by jiemeb** [comment]
Hi,

I have two display with resolution of 1680x1030 each.

the width of windows don't change anything. but the height yes

on screen resolution of 1440x900 all is fine.

the problem appear when on tab monitor vertical scroll bar on the right 
disappear.


Regards

Jean-Marie


Le 05/01/2023 à 14:07, Aldo Hoeben a écrit :
